packages/fake-data/src/fakeUsers.ts

```diff
@@ -121,6 +121,7 @@ const generateUser = (
 
     // Gov info
     isGovEmployee: faker.datatype.boolean(),
+    workEmail: faker.internet.email(),
```

Comment:
Maybe more realistic to do something like faker.userName() + "@gc.ca"

Reply:
Same for L272

Reply:
Great idea, updated in [improve fake work email](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12962/commits/2307d8f213863b7040510592055cd1080807eca0)